src/Clover/Provider.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace SocialiteProviders\Clover;
+
+use GuzzleHttp\RequestOptions;
+use Illuminate\Support\Arr;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    const IDENTIFIER = 'CLOVER';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = [''];
```

Comment:
why empty string here? can we just leave the array empty?

Reply:
I think I borrowed this example from another existing provider 🤷🏻‍♂️